### DISCUSSION TITLE
Support join by multi fields in transformation

### DISCUSSION THREAD

**Discussion by KippaZou** [grafana/grafana#38390] [Dashboards, Panels & Visualizations]
**What would you like to be added**:

Outer join in transformation supports join by multi fields.

**Why is this needed**:

For example, I have several queries A with columns (1, 2, 3), B with columns (1, 2, 4)  and C with columns (1, 2, 5). I wan't to join these queries by fields 1 and 2. But outer join in transformation only supports joining by one field. When add more transformations, the former will be covered.

**Comment by pkolyvas** [comment]
We're in the process of moving Grafana feature requests to [GitHub Discussions](https://community.grafana.com/t/were-moving-grafana-feature-requests-to-github-discussions/50341/3). This issue is being moved to [Discussions](https://github.com/grafana/grafana/discussions) or has been moved from our issues list, depending on where you're seeing this message. 

 We hope to use the discussion thread to continue building use cases, and additional evidence based on engagement/voting, in order to help us prioritize this feature request later on down the road. 

As with all things Grafana, we value your input and participation in this project and appreciate your patience during this transitional period. Looking forward to the discussion!

**Comment by imReker** [comment]
Another use case:
Metric1: A, B, value
Metric2: A, value
Metric3: B, value

Metric1A - Metric2A and Metric1B - Metric3B are 1:1 mapping.

Currently there is no way to join 3 Metrics to A, B, C, valueA, valueB, valueC.

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). We are migrating this discussion to an issue and closing the discussion. The issue is #82967. Feel free to continue the discussion around this there. Thank you!
